include/prereq-build.mk

```diff
@@ -47,8 +47,8 @@ $(eval $(call TestHostCommand,working-gcc, \
 
 $(eval $(call SetupHostCommand,g++, \
 	Please install the GNU C++ Compiler (g++) 4.8 or later \
-	$(CXX) -dumpversion | grep -E '^(4\.[8-9]|[5-9]\.?)', \
-	g++ -dumpversion | grep -E '^(4\.[8-9]|[5-9]\.?)', \
+	$(CXX) -dumpversion | grep -E '^(4\.[8-9]|[5-9]|10\.?)', \
+	g++ -dumpversion | grep -E '^(4\.[8-9]|[5-9]|10\.?)', \
```

Comment:
Why not something like this instead?
```g++ -dumpversion | grep -E '^(4\.[8-9]|[5-9]\.?|1[0-9]\.?)', \```
This way compatibility for point versions from 5.y to 9.y is retained, while adding support for 10, 11,... up to 19.y.
This should be done also for gcc.